lib/busted/profiler.rb

```diff
@@ -2,6 +2,7 @@
 require "busted/countable"
 require "busted/tracer"
 require "busted/traceable"
+require "busted/middleware/busted_tracer"
```

Comment:
Oh, I realized that we probably don't need to require the middleware in all cases. Do you think it would be worth it try to load it dynamically?